### PR TITLE
Add support with Github actions

### PR DESCRIPTION
**Pull request by gabechu**
### Description
No more travis and let's replace with Github actions.

### CODE REVIEW

**Comment by gabechu** [comment]
Regarding to the CI failure, I'm getting a `FileNotFoundError` when installing `pii_recognition` project.

File: .github/workflows/python.yml
Comment (gabechu) [inline .github/workflows/python.yml:37]: The cache key includes the Python version, otherwise bumping Python versions would cause re-installation of dependencies.

File: .github/workflows/python.yml
Comment (zendesk-cirode) [inline .github/workflows/python.yml:18]: ooo! using the pyproject.toml file as config? I like it!

File: .github/workflows/python.yml
Comment (zendesk-cirode) [reply .github/workflows/python.yml:37]: No it wouldn't @gabechu  as the python version is in the poetry lock so it would invalidate the cache. There's no real need to add this I dont think

**Review by zendesk-cirode — APPROVED** [review]

File: .github/workflows/python.yml
Comment (gabechu) [reply .github/workflows/python.yml:37]: I found that both caching and installation were triggered when change Python version `3.7.7` to `3.7` (Github fetched `3.7.9`) with `{{ runner.os }}-cache-dir-${{ hashFiles('poetry.lock') }}`. I am not sure what caused that, just suspecting maybe the version stuff.

image

Comment (gabechu) [reply .github/workflows/python.yml:37]: Also, does `poetry.lock` captures which Python we've installed? A quick search shows Python version is defined in `pyproject.toml` but not in `poetry.lock` (probably I missed it).

**Comment by gabechu** [comment]
At this point, nothing has changed after Chris's approval, all the commits were experimenting how cache works in Github Actions and they have been reverted. Gabe is going to merge this PR as he's been clicking with Travis for 2hrs and still can't get them all green!

p.s. we can always open a new PR on whether we want to add python versions as part of cache key.

**Pull request merged**
